### ISSUE TITLE
Add Learning Rate Scheduler

### ISSUE THREAD

**Issue by Tanyajain2006** [Tanyajain2006/Waste-Segregation#9]
Reduce the learning rate dynamically if validation loss plateaus.

**Comment by SanviTripathi** [comment]
hello!I 
would like to work on this issue .Could you plz assign it to me!
